webpack.config.ts

```diff
@@ -215,6 +215,7 @@ module.exports = {
     new HtmlWebpackPlugin({
       chunks: ['community_dashboard'],
       filename: 'community-dashboard-page.mainpage.html',
```

Comment:
@vojtechjelinek should we add a validator for our webpack.config.ts to ensure no missing keys etc.? It could be part of the lint checks.

Reply:
We can, I currently don't have the capacity to do so.

Reply:
Issue created #7414